src/test/java/edu/ucsb/courses/controllers/StatisticsControllerTests.java

```diff
@@ -33,6 +36,7 @@
 import edu.ucsb.courses.documents.statistics.AggregateStatistics;
 import edu.ucsb.courses.repositories.ArchivedCourseRepository;
 import edu.ucsb.courses.documents.statistics.OpenCourse;
+import edu.ucsb.courses.documents.statistics.SingleCourseSearch;
```

Comment:
This is an unused import, we should remove.

Reply:
We removed the unused import

Reply:
This has been fixed.